pep-0601.txt

```diff
@@ -207,30 +207,30 @@ How to Teach This
 
 This feature is very rarely used so forbidding it will likely only impact
 advanced users, not beginners and probably not any existing teaching
-material.  Since this is the removal of a feature teaching users will be 
-one by the raising of a SyntaxError if/when the forbidden feature is used.
+material.  Since this is the removal of a feature teaching users will be
+one by the raising of a ``SyntaxError`` if/when the forbidden feature is used.
 
 Reference Implementation
 ========================
 
 There is currently no reference implementation, although the way continue
-is currently handled in a finally (raising a SyntaxError) can be extended
-to return and break.
+is currently handled in a ``finally`` (raising a ``SyntaxError``) can be extended
+to ``return`` and ``break``.
 
 References
 ==========
 
-.. [1] https://bugs.python.org/issue37830
+.. [1] https://github.com/python/cpython/issues/82011
 
 .. [2] https://stackoverflow.com/questions/48088/returning-from-a-finally-block-in-java
 
 .. [3] https://web.archive.org/web/20070922061412/http://weblogs.java.net/blog/staufferjames/archive/2007/06/_dont_return_in.html
 
 .. [4] https://wiki.sei.cmu.edu/confluence/display/java/ERR04-J.+Do+not+complete+abruptly+from+a+finally+block
 
-.. [5] https://github.com/rubocop-hq/rubocop/issues/5949
+.. [5] https://github.com/rubocop/rubocop/issues/5949
 
-.. [6] https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/EnsureReturn
+.. [6] https://web.archive.org/web/20210622135940/https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/EnsureReturn
```

Comment:
```suggestion
.. [6] https://www.rubydoc.info/gems/rubocop/0.74.0/RuboCop/Cop/Lint/EnsureReturn
```

Instead of falling back to the Wayback Machine, we could just pin the version that was current when this link was added